data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,73 @@ mission "Station Exploration Abandon Ship"
 			label fine
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
+mission "Hai Pilgrimage"
+	name "Hai spiritual leader to <planet>"
+	description "Bring a Hai spiritual leader to <destination>. He will pay you <payment>."
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	destination
+		attributes "unfettered"
+	passengers 1
+	to offer
+		random < 30
+		has "outfit: Quantum Keystone"
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`It is an unusually foggy day on <source>; the air is so thick with moisture that you cannot see more than a few paces in front of you. Perhaps that's why you're so startled when you turn around and see a Hai standing right next to you. He is so close you can see dewdrops on his whiskers, and you immediately notice the prominent jewelry that he is wearing. He appears to be quite old, judging by his stooped-over posture and the white fur around his muzzle.`
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise. Are you willing and able to take me to <destination>? I can pay <payment>."`
```

Comment:
Isnt this a bit too direct? Mb he's like that tho

Reply:
Inserted a pause before he asks you for transport.